Tasks/AndroidSigningV2/package.json

```diff
@@ -17,8 +17,12 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@types/node": "^6.0.0",
-    "azure-pipelines-task-lib": "^2.8.0",
-    "azure-pipelines-tasks-securefiles-common": "1.0.0"
+    "@types/node": "^10.17.0",
+    "@types/mocha": "^5.2.7",
+    "azure-pipelines-task-lib": "^3.0.6-preview",
```

Comment:
Nit, +@egor-bryzgalov just to discuss one more time. Should we specify task-lib version with **.0** at the end?

> Upgrade azure-pipelines-task-lib to 3.0.6-preview.0 in package.json dependencies.

"^3.0.6-preview" syntax will also work (until someone uploads exactly this version (3.0.6-preview) to npm)

Reply:
@max-zaytsev That's a good point! I think it would be safer to specify `.0`.